aten/src/ATen/native/cuda/Sort.cu

```diff
@@ -254,6 +261,13 @@ inline void segmented_sort_pairs_by_full_sort(
   const int64_t nsegments, const int64_t nsort, const int64_t n, const bool descending,
   const scalar_t *const self_ptr, scalar_t *const values_ptr, int64_t *const indices_ptr
 ) {
+  if (nsegments == 1) {
+    auto indices = at::arange(nsort, TensorOptions().device(at::kCUDA).dtype(at::kLong));
```

Comment:
`at::arange` requires `Tensor.h` to be included, so you can't use it with `TORCH_ASSERT_NO_OPERATORS`. You could always use a small custom kernel instead.

Reply:
Hm, it looks like Embedding.cu and BatchLinearAlgebra.cpp are using it? Oh, no, they have TORCH_ASSERT_ONLY_METHOD_OPERATORS, ok, that makes sense.

Reply:
Yeah, `Sort.cu` is actually a lot slower to compile because of the `cub` usage so it uses the stricter macro.